server/webhook.go

```diff
@@ -287,6 +295,8 @@ func (p *parsedJIRAWebhook) fromChangeLog(issue string) (string, string) {
 	for _, item := range p.ChangeLog.Items {
 		to := item.ToString
 		from := item.FromString
+		lto := strings.ToLower(item.ToString)
```

Comment:
nit: 1/5 I'm worried this will slow down reading the code, can we use longer variable names and prioritize readability over typing speed? (it took me a second to understand the `l` meant lower)

Reply:
It's gone now. 2/5 https://github.com/golang/go/wiki/CodeReviewComments#variable-names in this case, though `lto` may not have been the best choice, I agree.